packages/manager/apps/dedicated/client/app/dedicatedCloud/drpDatacenterSelection/dedicatedCloud-drpDatacenterSelection.html

```diff
@@ -0,0 +1,18 @@
+<oui-modal data-heading="{{:: 'dedicatedCloud_drp_datacenter_selection_title' | translate }}"
+           data-primary-action="$ctrl.selectDatacenter($ctrl.datacenter.id)"
+           data-primary-label="{{:: 'dedicatedCloud_drp_datacenter_selection_action' | translate }}"
+           data-primary-disabled="!$ctrl.datacenter"
+           data-secondary-action="$ctrl.goToPccDashboard()"
+           data-secondary-label="{{:: 'common_cancel' | translate }}"
+           data-on-dismiss="$ctrl.goToPccDashboard()"
+           data-loading="$ctrl.isRedirecting">
+    <form novalidate name="$ctrl.datacenterSelection">
```

Comment:
The `<form>` element is supposed to wrap the `<oui-modal>` component I guess (see: https://github.com/ovh-ux/ovh-ui-angular/blob/master/packages/oui-modal/src/modal.html#L36)